genai-perf/genai_perf/parser.py

```diff
@@ -1056,11 +1066,50 @@ def _add_profile_args(parser):
 def _add_session_args(parser):
     session_group = parser.add_argument_group("Session")
 
+    session_load_management_group = session_group.add_mutually_exclusive_group(
```

Comment:
I might create a separate load management category, since there should only be one of --request-concurrency, --request-rate, --session-concurrency, and --session-rate ever selected. That will improve documentation and error checking, plus it will be more native to ArgParse.

Reply:
Agree 100%. Lets get this in, but we will need to expand the load_management groups soon.